src/site/content/en/handbook/self-assessment-components/self-assessment.assess.yml

```diff
@@ -83,17 +83,17 @@ questions:
     stem: "Which **two** Sesame Street muppets are clearly the best?"
     columns: true
     options:
-      - content: "![Elmo](./elmo.jpg)"
+      - content: "![Elmo](image/tcFciHGuF3MxnTr1y5ue01OGLBn2/ng9hB0jbzGXy0MY7upMU.jpg)"
```

Comment:
does this get transformed into a full image path?

Reply:
Yes, the Assessment shortcode will go through all the img tags and convert them to use imgix.